front/app/components/admin/Graphs/LineChart/index.tsx

```diff
@@ -86,7 +89,7 @@ const LineChart = <Row,>({
       onUpdateLegendDimensions={setLegendDimensions}
     >
       <RechartsLineChart
-        data={data}
+        data={isNilOrError(data) ? undefined : data}
         margin={parseMargin(
```

Comment:
In some cases (E.g. Community monitor empty state), we might want to show an empty graph when we don't have any data yet.